Lib/venv/scripts/nt/activate.bat

```diff
@@ -1,13 +1,12 @@
 @echo off
 
-rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
-    set "_OLD_CODEPAGE=%%a"
-)
+rem This file is UTF-8 encoded, so we need to update the current code page while executing it.
+for /f %%a in ('__VENV_PYTHON__ -s -S -E -c "from ctypes import windll; print(windll.kernel32.GetConsoleOutputCP());"') do (set "_OLD_CODEPAGE=%%a")
```

Comment:
We can't use `__VENV_PYTHON__` since it's UTF-8. Go back to using `%dp0python.exe`. 

The command is shorter without the `from` import, e.g. `import ctypes; print(ctypes.windll.kernel32.GetConsoleOutputCP())`.